### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by richpryce**
## What does this change?

Update local development docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -417,6 +418,7 @@ to step 4.
 - Once you have removed the extraneous node_modules, Build the host project you are testing with:
    ```
    yarn dev
+   cd build
```
Comment (Dala0) [inline README.md:421]: Please remove this line as it is not required when using sbt. If you really want to including it, please rewrite the line below to include it.

**Review by Dala0 — CHANGES_REQUESTED** [review]
Please remove or rewrite line 421.

File: README.md (hunk 1 above)
Comment (Dala0) [reply README.md:421]: Actually not all the software using Docker to build directly has a build folder, so best to just remove it.

Comment (richpryce) [reply README.md]: Good point, thanks.

Comment (Dala0) [reply README.md]: You are welcome.

**Pull request merged**